lib/utils.js

```diff
@@ -21,3 +21,13 @@ export function deprecated (fn, message) {
 
   return newFn
 }
+
+export function printAndExit (message, code = -1) {
```

Comment:
but I wonder why we use `-1` instead of `1`. I don't know what minus numbers mean for exit code.